src/main/java/net/glowstone/GlowServer.java

```diff
@@ -2537,7 +2537,7 @@ public String getConsoleLogDateFormat() {
     /**
      * Gets the server type.
      *
-     * <p>Currently, this value is set to {@code VANILLA}.
+     * <p>Currently, this value is set to {@code VANILLA} and is not localized.
```

Comment:
I don't like this doc addition. It's like specifying an enum's members are not localized, since this is essentially what the server type string is. It can be `BUKKIT` for Bukkit+Forge modded servers, `VANILLA` for non-Forge servers or `FML` for Forge servers. If you want, you can add that to the documentation.

Reply:
Rewritten.

Reply:
Note that StatusRequestHandler includes the server type in its output, in a form that AFAICT is likely to be displayed to players, so the explanation isn't as simple as for an enum value's name.

Reply:
Yes, the client takes the server type and then displays a name according to that type. For example, `BUKKIT` -> `Bukkit modded server` IIRC.

Reply:
Rewritten accordingly.

Reply:
And I assume that localizing to client-side "Bukkit modded server" implies a modded client? I don't remember Mojang officially acknowledging the existence of Bukkit.

Reply:
Yes, this is not part of the standard protocol, it's for Forge clients.